src/api/post.js

```diff
@@ -0,0 +1,21 @@
+import { client } from "./httpClient";
+
+export const getUserPosts = (userId) => {
+  return client.get(`/posts?userId=${userId}`);
+};
+
+export const getPosts = () => {
+  return client.get("/posts");
+};
+
+export const createPosts = ({ body, title, userId }) => {
+  return client.post("/posts", { body, title, userId });
+};
+
+export const updatePost = ({ body, title, postId }) => {
+  return client.patch(`/posts/${postId}`, { body, title });
+};
+
+export const deletePost = (postId) => {
+  return client.delete(`https://mate.academy/students-api/posts/${postId}`);
```

Comment:
You are using a full URL in the `deletePost` method, while in other methods you use relative URLs. This can lead to errors if the base URL changes. It would be better to keep consistency and use a relative URL here as well, like in other methods.